tezos/context/src/tezedge_context.rs

```diff
@@ -438,29 +489,29 @@ impl TezedgeIndex {
     fn get_key_values_from_tree_recursively(
```

Comment:
This one is missing documentation. Also, I guess these methods that don't return the value but instead modify an input could use another naming convention that is different from what is used elsewhere. Maybe `collect_` ?